noetic/distribution.yaml

```diff
@@ -1333,6 +1333,31 @@ repositories:
       url: https://bitbucket.org/dataspeedinc/lusb.git
       version: master
     status: developed
+  marker_msgs:
+    doc:
+      type: git
+      url: https://github.com/tuw-robotics/marker_msgs.git
+      version: noetic-devel
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
```

Comment:
Whoops, didn't notice this before. This should be `release/noetic/{package}/{version}` instead of `melodic`

https://github.com/tuw-robotics/marker_msgs-release/tree/release/noetic/marker_msgs/0.0.6-1

Reply:
@maxbader friendly ping :)